src/Atmos/Model/AtmosModel.jl

```diff
@@ -64,32 +65,67 @@ import ..Courant: advective_courant, nondiffusive_courant, diffusive_courant
 """
     AtmosModel <: BalanceLaw
 
-A `BalanceLaw` for atmosphere modeling.
+A `BalanceLaw` for atmosphere modeling. 
+Users may over-ride prescribed default values for each field.
 
 # Usage
 
-    AtmosModel(orientation, ref_state, turbulence, hyperdiffusion, moisture, radiation, source, tracers,
-               boundarycondition, init_state)
+    AtmosModel(
+        param_set, 
+        orientation, 
+        ref_state, 
+        turbulence, 
+        hyperdiffusion, 
+        moisture, 
+        radiation, 
+        source, 
+        tracers,
+        boundarycondition, 
+        init_state
+    )
+
 
+# Fields
+$(DocStringExtensions.FIELDS)
 """
 struct AtmosModel{FT, PS, O, RS, T, HD, M, P, R, S, TR, BC, IS, DC} <:
        BalanceLaw
+    "Parameter Set (e.g. Planet Parameters, coupling with CES package)"
     param_set::PS
+    "Orientation (Flat(LES in a box) or Spherical(GCM))"
```

Comment:
Since there's only two, maybe we can make use of [`Flat`](@ref) here?

Reply:
Oh, I just realized that didn’t render the way I meant it to:
```
[`Flat`] (@ref)
```

Reply:
But without the space between ] and (

Reply:
orientation docs haven't been set up yet, so this would be an issue ?